content/docs/command-reference/plots/modify.md

```diff
@@ -131,3 +135,27 @@ be changed in many cases. A simple command changes the template:
 ```dvc
 $ dvc plots modify classes.csv --template confusion
 ```
+
+## Example: Convert any output into a plot
+
+We can use `dvc plots modify` to make an <abbr>output</abbr> or simple metrics
+file of a stage into a plot. Lets take an example `evaluate` stage which has
+`logs.csv` as output. To change the output file `logs.csv` into a plot run:
+
+```dvc
+$ dvc plots modify logs.csv
+```
+
+Changes in `dvc.yaml` shows `logs.csv` is now a plot:
+
+```git
```

Comment:
This is great but now that I read it vs. the paragraph in the description (which has all the info you need, really) it looks like overkill to have an entire example for this... Not sure.

I guess it doesn't hurt? But let's try to condense it more: 1 short paragraph (2 sentences preferrably), the `dvc` code block, and the `git` code block. There's a challenge for you @imhardikj !

Reply:
> let's try to condense paragraph more

Updated will much simpler info

Reply:
Yep, almost perfect! (missing a "the")